client/src/utils/CastVoteRecordFiles.ts

```diff
@@ -103,6 +103,35 @@ export default class CastVoteRecordFiles {
     this.allCastVoteRecords = castVoteRecords
   }
 
+  public export() {
+    return JSON.stringify({
+      signatures: JSON.stringify([...this.signatures]),
+      files: JSON.stringify([...this.files]),
+      duplicateFilenames: JSON.stringify([...this.duplicateFilenames]),
+      parseFailedFilenames: JSON.stringify([...this.parseFailedFilenames]),
+      allCastVoteRecords: JSON.stringify(
+        Array.from(this.allCastVoteRecords.entries())
+      ),
+    })
+  }
+
+  public import(stringifiedCVRFiles: string) {
+    const {
+      signatures,
+      files,
+      duplicateFilenames,
+      parseFailedFilenames,
+      allCastVoteRecords,
+    } = JSON.parse(stringifiedCVRFiles)
```

Comment:
Would love to have some validation around these at some point. See votingworks/ballot-encoder#20.

Reply:
👍 Me too. Will need a little guidance with this.